operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -3047,6 +3047,77 @@ minio:
   # Changed the mc config path to '/tmp' from '/etc' as '/etc' is only writable by root and OpenShift will not permit this.
   configPathmc: "/tmp/minio/mc/"
 
+kafka:
+  # -- Enable Kafka for ingest-storage architecture (demo purposes only)
+  enabled: false
+
+  image:
+    # -- The container image registry
+    registry: docker.io
+    # -- The container image repository
+    repository: apache/kafka-native
+    # -- The container image tag
+    tag: 4.0.0
+    # -- The container image pull policy
+    pullPolicy: IfNotPresent
+
+  # -- Kafka service port
+  service:
+    port: 9092
+    controllerPort: 9093
+    # -- Service type
+    type: ClusterIP
+    # -- Service annotations
+    annotations: {}
+    # -- Service labels
+    labels: {}
+  # -- Kafka persistence configuration
+  persistence:
+    enabled: true
+    size: 5Gi
+    storageClassName: ""
+  # -- Resource requests and limits for Kafka
+  resources:
+    requests:
+      cpu: 100m
```

Comment:
This CPU and memory seems too low for a Java process. I'd err on the side of giving it too many resources to avoid having people run into weird issues where it doesn't start. Maybe 2 CPUs and a few gigs of memory?

Reply:
I actually run a single-node java-based (non-native binary) Kafka in a homelab VM for a year. It peaks at 1GB memory 👍 

```
root@kafka-1:~# systemctl status kafka.service

   Main PID: 268 (java)
      Tasks: 94 (limit: 34661)
     Memory: 1.0G (peak: 1.1G)
        CPU: 1h 11min 23.069s
     CGroup: /system.slice/kafka.service
```